patches/tModLoader/Terraria/ModLoader/Engine/GoGVerifier.cs

```diff
@@ -0,0 +1,127 @@
+﻿using ReLogic.OS;
+using Steamworks;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Security.Cryptography;
+using Terraria.Social;
+
+namespace Terraria.ModLoader.Engine
+{
+	internal static class GoGVerifier
```

Comment:
no longer checking the content dir for valid installs? I'd rather you revert this